### PR TITLE
Pylint - Docstring in Alignak

### PR DESCRIPTION
**Pull request by Seb-Solon**
Docstring are now ok for all file (except the one we skipped).

Enforced by travis.

If you edit docstring you will have to regenerate the rst in /doc

### CODE REVIEW

File: alignak/action.py [hunk 1]
```diff
@@ -79,8 +83,13 @@
                    '|', '{', '}', ';', '<', '>', '?', '`')
 
 
-# Try to read a fd in a non blocking mode
 def no_block_read(output):
+    """Try to read a fd in a non blocking mode
```
Comment (ddurieux) [inline alignak/action.py]: Need to not use abbreviation, what is 'fd'? 
Need to use the complete word else it's not easy to understand

File: alignak/action.py [hunk 2]
```diff
@@ -79,8 +83,13 @@
...
+
+    :param output: file or socket to read from
+    :type: file
+    :return: data read from fd
```
Comment (ddurieux) [inline alignak/action.py]: Add a line with type of return
```
:rtype: str
```

File: alignak/action.py [hunk 3]
```diff
@@ -99,20 +108,22 @@ class __Action(object):
...
     id = 0
 
-    # Ok when we load a previous created element, we should
-    # not start at 0 for new object, so we must raise the Action.id
-    # if need
     @staticmethod
     def assume_at_least_id(_id):
+        """Set Action.id to the maximum of itself and _id
+
+        :param _id: action id to compare (from a previous run usually)
```
Comment (ddurieux) [inline alignak/action.py]: Add a line with the type of _id

File: alignak/action.py [hunk 4]
```diff
@@ -155,6 +165,14 @@ def execute(self):
 
 
     def get_outputs(self, out, max_plugins_output_length):
+        """Get outputs from single output (splt perfdata etc)
```
Comment (ddurieux) [inline alignak/action.py]: Fix typo: splt by split

File: alignak/action.py [hunk 5]
```diff
@@ -155,6 +165,14 @@ def execute(self):
...
+        Edit output, perf_data and long_output attributes
+
+        :param out: output data of a check
```
Comment (ddurieux) [inline alignak/action.py]: Add the type of 'out' param

**Comment by ddurieux** [comment]
I have begun to review, but to be sure with docstring for all files, when have no return, we keen the ':return:' without value or delete it?

**Comment by Seb-Solon** [comment]
No return == None. We could put None. For type of other params sometimes its obvious ans sometime I dont know :)

**Comment by Seb-Solon** [comment]
Related to #4, #87, #262 and naparuba/shinken#1393 .

**Pull request merged**